runtime/src/main/kotlin/kotlin/text/Strings.kt

```diff
@@ -241,7 +241,6 @@ public actual fun String(chars: CharArray, offset: Int, length: Int): String {
  * Concatenates characters in this [CharArray] into a String.
  */
 @SinceKotlin("1.3")
-@ExperimentalStdlibApi
```

Comment:
Do we still need ExperimentalStdlibApi in build file then?

Reply:
This opt-in was added to compile the test source set only.
We plan to ship more such API in stdlib-common in the near future, so having that opt-in ready will make native tests compile without errors then.